src/monodroid/jni/dylib-mono.h

```diff
@@ -109,6 +109,36 @@ typedef struct {
 	const unsigned int size;
 } MonoBundledAssembly;
 
+// Keep this in sync with MonoObject in object.h
+typedef struct {
+	// MonoObject 2 pointers
+	void *_pad_mono_object[2];
+} MonoObject;
+
+// Keep this in sync with MonoAppDomainSetup in domains-internal.h
+typedef struct {
+	MonoObject object;
+	// MonoAppDomainSetup unused entries
+	void *_pad_mono_appdomainsetup[9];
+
+	MonoString *shadow_copy_files;
+} MonoAppDomainSetup;
+
+// Keep this in sync with MonoRealProxy in object-internals.h
+typedef struct {
+	MonoObject object;
+	// MonoRealProxy ununused entries
+	void *_pad_mono_realproxy[3];
+
+	int32_t target_domain_id;
+} MonoRealProxy;
```

Comment:
I don't see why this needs a "real" declaration either: the struct members aren't used.